lib/deaf_grandma.rb

```diff
@@ -14,13 +14,21 @@ def run!
     loop do
       user_input = get_user_input
       p speak(user_input)
+      break if @bye_counter == 3
```

Comment:
Since `@bye_counter` is compared with `3` in two places, consider extracting a method to combine the duplication. In an actual application, this sort of duplication might lead to a bug when one is not kept in sync with the other.